src/components/InjectedComponents/PostDialog.tsx

```diff
@@ -147,11 +147,15 @@ export function PostDialogUI(props: PostDialogUIProps) {
                     <DialogContent className={classes.content}>
                         {withMetadata(props.postContent.meta, 'com.maskbook.red_packet:1', r => (
                             <Chip
-                                onDelete={() => {
+                                onDelete={async () => {
                                     const ref = getActivatedUI().typedMessageMetadata
                                     const next = new Map(ref.value.entries())
                                     next.delete('com.maskbook.red_packet:1')
                                     ref.value = next
+                                    if (props.onShareToEveryoneChanged) {
+                                        await sleep(300)
+                                        props.onShareToEveryoneChanged(false)
```

Comment:
why add a delay here?

Reply:
For resetting `Share to Everyone` clip to unselect state. In addition, `ref.value = next` is async so wait a while for ensure really reseted.

Reply:
maybe add a comment here